src/utils/embeds.ts

```diff
@@ -0,0 +1 @@
+export const EMBED_COLOUR = '#0099ff';
```

Comment:
What colour is this tho :O 
Is it Codey-Fuzz-Blue ?

Reply:
no idea I just used whatever @alexz429 used before

Reply:
Its def not the right blue, might have to ask Design on that one.